TW 264 Refresh APIMatic page (#3873)
postmanlabs/postman-docs: src/pages/docs/integrations/available-integrations/apimatic.md

    url: "https://blog.postman.com/new-postman-pro-integration-apimatics-api-transformer/"
---

> Postman's APIMatic integration converts your Postman collections to any major API description format, including Swagger, RAML, and API Blueprint, and saves the resulting file on GitHub.

Reviewer: A couple of things:
* don't say 'any' because I'm sure there will be other major formats not supported. Maybe "an API description format, such as... "? 
* Instead of saves, say "backs up"
* Maybe "periodically backs up" to show it's not a one-time thing?

(Also, don't forget to increment the date in the frontmatter.)

Author: Good call(s)! Updated in [214789b](https://github.com/postmanlabs/postman-docs/pull/3873/commits/214789bf3c46d4c4714c5689860fb529f2f12860)

## Prerequisites

To set up an APIMatic integration with Postman, you will need:

* An [APIMatic account](https://www.apimatic.io/account/register)
* An [APIMatic integration key](https://docs.apimatic.io/manage-apis/get-api-keys/)
* A [GitHub account](https://github.com/)
* A GitHub repository in which to save your collection

## Configuring an APIMatic integration

To back up a collection periodically to a GitHub repository:

1. On your Postman **[Home](https://go.postman.co/home)** page, select **[Integrations](https://go.postman.co/integrations)**.
1. Type **APIMatic** into the search bar and select the result.

    ![browse and select APIMatic](https://assets.postman.com/postman-docs/apimatic-browse-all.jpg)

1. Select **Add Integration**.

> 1. Your browser will open a new Authorize Postman page that gives Postman access to your GitHub repositories. Select Authorize postmanlabs. Once the authorization process is complete, the page will tell you to close the tab and continue in Postman.

Reviewer: ```suggestion
1. Your browser will open a new **Authorize Postman** page that gives Postman access to your GitHub repositories. Select **Authorize postmanlabs**. Once the authorization process is complete, the page will tell you to close the tab and continue in Postman.
```

Author: Oh boy 🙈  Thank you for catching those!

    > If you are not signed in to GitHub, you will be prompted to log in.

1. In Postman, you will be prompted to enter the following information to create an integration:

>     * Your [APIMatic integration key](https://docs.apimatic.io/manage-apis/get-api-keys/).

Reviewer 2: This list of fields is rather long and the bullets are fairly undifferentiated. Maybe consider using bold UI terms where appropriate to help orient the user as they go through the list? For examples, see the New Relic or Slack integration pages.

Author: Good call! Updated in [1213b0e](https://github.com/postmanlabs/postman-docs/pull/3873/commits/1213b0e33fdf0f8df9ae7b4d669318c9f710bb94)

    * A nickname for the integration.
    * The Postman workspace that the collection you want to back up belongs to.
    * The Postman collection you want to back up.
    * The GitHub repository your collection will be saved to.
    * The directory in the GitHub repository in which to save your collection. If a directory with that name does not exist, it will be created when you add the integration.
    * A filename for the backup file.
    * The format in which you want to save the collection.
    * The name of the GitHub branch to save the collection to. This branch must already exist, and the integration process will fail if you specify an invalid branch name.
1. Select **Add Integration**.

    <img alt="APIMatic integration form" src="https://assets.postman.com/postman-docs/apimatic-save-config-v9.jpg" width="400px"/>

> Once the integration has been created, click on it to view details including the run log.

Reviewer: click/select

> Once the integration has been created, click on it to view details including the run log.

Author: ```suggestion
Once the integration has been created, select it to view details about the integration including its run log.
```

> Once the integration has been created, click on it to view details including the run log.

Reviewer 2: ```suggestion
Once the integration has been created, select it to view details including the run log.
```
